build.sbt

```diff
@@ -2,10 +2,11 @@ import sbtrelease.ReleaseStateTransformations._
 
 name:= "content-api-firehose-client"
 organization := "com.gu"
-scalaVersion := "2.12.8"
-crossScalaVersions := Seq("2.11.12", scalaVersion.value)
-scalacOptions ++= Seq("-feature", "-deprecation", "-unchecked", "-target:jvm-1.8", "-Xfatal-warnings")
+scalaVersion := "2.12.17"
+crossScalaVersions := Seq(scalaVersion.value, "2.13.8")
+scalacOptions ++= Seq("-feature", "-deprecation", "-unchecked", "-release:8", "-Xfatal-warnings")
 Compile / doc / scalacOptions  := Nil
+credentials += Credentials(Path.userHome / ".sbt" / "sonatype_credentials")
```

Comment:
Is this necessary or just while we're testing it locally?

Reply:
👍